lib/triggers/messageIntervalTrigger.js

```diff
@@ -24,15 +24,15 @@ exports.create = function(name, chatBot, options) {
 };
 
 MessageIntervalTrigger.prototype._onLoad = function() {
-	setInterval(this._send, this.options.interval);
+	setInterval(function(){this._send(this)}, this.options.interval);
 }
 
-MessageIntervalTrigger.prototype._send = function() {
-	try {
-		if(this.options.destination && this.options.message && this.chatBot.connected) {
-			this._sendMessageAfterDelay(this.options.destination, this.options.message);
+MessageIntervalTrigger.prototype._send = function(that) {
+		try {
```

Comment:
Can you remember why we needed to do this? I can't, and I can't think of a reason why these lines should be removed. I recall this function causing the bot to crash when people had triggers sending messages as soon as the bot was started (ie, before it was connected, hence the check for `this.chatBot.connected`). The try block should deal with it, but it would also cause unnecessary errors in the log/console. I should definitely be logging *something* rather than letting the messages go nowhere, but dropping the check entirely seems the wrong way to do it.